components/Turnstile.vue

```diff
@@ -0,0 +1,65 @@
+<template>
+  <VueTurnstile
+    v-model="token"
+    :reset-interval="resetInterval"
+    :size="size"
+    :theme="theme"
+    :action="action"
+    :appearance="appearance"
+    :render-on-mount="renderOnMount"
+    :site-key="siteKey" />
+</template>
+
+<script setup>
+/*!
+  * Copyright (c) 2023 Digital Bazaar, Inc. All rights reserved.
+  */
+import {config} from '@bedrock/web';
+import {defineEmits, defineProps, ref, watch} from 'vue';
```

Comment:
Is the linter working properly? This should have errored since multiple syntax is expected before single syntax.